config/cloud-config/node-config-template.yaml

```diff
@@ -73,19 +73,28 @@ write_files:
     permissions: 0755
     content: |
       #! /usr/bin/bash
-      
+      CALICO_VERSION=$(/usr/bin/etcdctl get /calico-install/version) >/dev/null 2>&1 || CALICO_VERSION="0.12.0"
```

Comment:
Will `etcdctl get` work like this on the node? For this guide, `etcd` is running on the master.  I'd expect that we need to pass `--endpoint` (or set the env variable) as described here: https://github.com/coreos/etcd/tree/master/etcdctl#endpoint